src/lightning/app/components/serve/python_server.py

```diff
@@ -293,7 +293,7 @@ def configure_layout(self) -> Optional["Frontend"]:
         try:
             from lightning_api_access import APIAccessFrontend
         except ModuleNotFoundError:
-            logger.warn("APIAccessFrontend not found. Please install lightning-api-access to enable the UI")
+            logger.warn("APIAccessFrontend not found. To enable the UI run: pip install lightning-api-access")
```

Comment:
```suggestion
            logger.warn("APIAccessFrontend not found. To enable the UI, run: pip install lightning-api-access")
```
Not sure, but I think the comma is missing.